clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/ops/job/CloneKubernetesJobAtomicOperation.groovy

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.deploy.ops.job
+
+import com.netflix.frigga.Names
+import com.netflix.spinnaker.clouddriver.data.task.Task
+import com.netflix.spinnaker.clouddriver.data.task.TaskRepository
+import com.netflix.spinnaker.clouddriver.deploy.DeploymentResult
+import com.netflix.spinnaker.clouddriver.kubernetes.api.KubernetesApiConverter
+import com.netflix.spinnaker.clouddriver.kubernetes.deploy.KubernetesUtil
+import com.netflix.spinnaker.clouddriver.kubernetes.deploy.description.job.CloneKubernetesJobAtomicOperationDescription
+import com.netflix.spinnaker.clouddriver.kubernetes.deploy.description.job.KubernetesJobRestartPolicy
+import com.netflix.spinnaker.clouddriver.kubernetes.deploy.exception.KubernetesResourceNotFoundException
+import com.netflix.spinnaker.clouddriver.orchestration.AtomicOperation
+import io.fabric8.kubernetes.api.model.extensions.Job
+
+class CloneKubernetesJobAtomicOperation implements AtomicOperation<DeploymentResult> {
+  private static final String BASE_PHASE = "CLONE_JOB"
+
+  CloneKubernetesJobAtomicOperation(CloneKubernetesJobAtomicOperationDescription description) {
+    this.description = description
+  }
+
+  private static Task getTask() {
+    TaskRepository.threadLocalTask.get()
+  }
+
+  CloneKubernetesJobAtomicOperationDescription description
+
+  /*
+   * curl -X POST -H "Content-Type: application/json" -d  '[ { "cloneJob": { "source": { "jobName": "kub-test-v000" }, "account":  "my-kubernetes-account" } } ]' localhost:7002/kubernetes/ops
+  */
+  @Override
+  DeploymentResult operate(List priorOutputs) {
+    CloneKubernetesJobAtomicOperationDescription newDescription = cloneAndOverrideDescription()
+
+    task.updateStatus BASE_PHASE, "Initializing copy of job for " +
+      "${description.source.jobName}..."
```

Comment:
It looks a little funny to have the string concatenated starting with an embedded value. I think it might look better to move the string containing the embedded values for this one, and for the next two, to the previous line. Even if the lines are a little long I think it will look better.